ara/cli/record.py

```diff
@@ -31,8 +31,8 @@ def get_parser(self, prog_name):
             help=("List records for the specified playbook"),
         )
         parser.add_argument(
-            "--key",
-            metavar="<key>",
+            "--recordkey",
+            metavar="<recordkey>",
```

Comment:
Could you explain this issue ? The name of the argument is key and it's what gets passed on to the query later on around line 90: https://github.com/ansible-community/ara/pull/416/files#diff-34f949807054e46caefa740bb513e676fe971ac4b9f71ac109bf432b5de4679bR90-R91

```
> ara record list --playbook 2931
+-----+--------------+------+----------+-----------------------------+
|  id | key          | type | playbook | updated                     |
+-----+--------------+------+----------+-----------------------------+
| 852 | list_as_text | text |     2931 | 2021-02-25T17:01:13.791119Z |
| 851 | somedict     | dict |     2931 | 2021-02-25T17:01:12.255127Z |
| 850 | somelist     | list |     2931 | 2021-02-25T17:01:10.659884Z |
| 849 | bar          | json |     2931 | 2021-02-25T17:01:09.095688Z |
| 848 | foo          | url  |     2931 | 2021-02-25T17:01:07.608241Z |
| 847 | notype       | text |     2931 | 2021-02-25T17:01:04.939126Z |
+-----+--------------+------+----------+-----------------------------+

> ara record list --playbook 2931 --key notype
+-----+--------+------+----------+-----------------------------+
|  id | key    | type | playbook | updated                     |
+-----+--------+------+----------+-----------------------------+
| 847 | notype | text |     2931 | 2021-02-25T17:01:04.939126Z |
+-----+--------+------+----------+-----------------------------+
```

Reply:
Oh, I was taking a quick glance at the code and missed your comment about it.

I'm not sure whether this is the right fix but we can investigate :p

Reply:
Thanks for sharing some practical examples. The right fix from my perspective would be to 

* either go with the proposed change of introducing `recordkey` and refactor all the code which relates to this
* Rename the clashing `key` argument used for the client certificate's private key to something like `privkey` or `certkey` and refactor those parts of the codebase.

Anyways, this should probably not be part of this PR but raised as separate issue and handled as separate PR - I'd rollback my changes to `ara/cli/record` for this PR in the meantime.

Reply:
This is a good catch :)

It's a breaking change to rename the arguments but it's also a new feature that hasn't yet been released so I will let it go without a deprecation cycle dance -- it's only in the master branch:
- https://github.com/ansible-community/ara/pull/392
- https://github.com/ansible-community/ara/pull/403

Since I'm the one who chose the initial argument names, I volunteer to replace ``--key``, ``--cert`` and ``--ca`` by ``--ssl-key``, ``--ssl-cert`` and ``--ssl-ca`` in another PR if no one gets to it first.

FYI @devon-mar